clusterloader2/testing/experimental/storage/pod-startup/volume-types/persistentvolume/gcp-csi-driver-stable.yaml

```diff
@@ -0,0 +1,343 @@
+apiVersion: v1
```

Comment:
@msau42 @davidz627 - can you please review this file - I don't know enough about what exactly is needed and how to configure it to make it work.

Reply:
Please see my latest comment. Tldr: it wont but raises an interesting problem.

Reply:
this looks like a copy paste of the `stable` driver :) Please comment at top of file a xref to the original files.

This is the correct set of yamls to deploy a driver into a cluster (along with the secret defined in that other file)